index.js

```diff
@@ -54,13 +54,21 @@ function inspectElements() {
 
 const addExtensionIfInstalled = (name, getPath) => {
 	const isExtensionInstalled = name => {
+		if (session.defaultSession.getAllExtensions) {
+			return {}.hasOwnProperty.call(session.defaultSession.getAllExtensions(), name);
+		}
+
 		return BrowserWindow.getDevToolsExtensions &&
```

Comment:
Add a TODO comment to remove this when targeting Electron 9. Same with the other place.